.github/workflows/buildTest.yml

```diff
@@ -10,9 +10,8 @@ jobs:
           mkdir lib
           cd lib/
           wget -O json.jar https://search.maven.org/remotecontent?filepath=org/json/json/20210307/json-20210307.jar
-          wget -O junit.jar https://repo1.maven.org/maven2/org/junit/platform/junit-platform-console-standalone/1.7.1/junit-platform-console-standalone-1.7.1-all.jar 
           cd ..
-      - run: javac -cp "lib/*" *.java
+      - run: javac -cp "lib/json.jar" *.java
```

Comment:
Tovabbra is ott van, hogy leforditod az osszes java fajlodat, igy a tesztjeidet is.